### DISCUSSION TITLE
How to get markers range?

### DISCUSSION THREAD

**Discussion by mateusz-glowinski** [gijzelaerr/python-snap7#405] [Q&A] [answered]
Hi,
Is there possibility to get map of merkers? Like to create a map of current markers bit values?

**Comment by mateusz-glowinski** [comment]
Bump

**Comment by gijzelaerr** [comment, marked as answer]
Closing as stale. The marker memory range depends on the specific PLC model. You can read marker bytes using `mb_read()` with the appropriate address range for your CPU.
